packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -156,6 +164,14 @@ export interface IProject extends IResource, iam.IGrantable, ec2.IConnectable {
   metricFailedBuilds(props?: cloudwatch.MetricOptions): cloudwatch.Metric;
 }
 
+/**
+ * The return value for the {@link Project} `enableBatchBuilds()` method.
+ */
+export interface IEnableBatchBuildsReturn {
```

Comment:
@skinny85 I named this this instead of `BatchBuildConfig` because I didn't want it to get confused with the actual `buildBatchConfig` in L1

Reply:
Meh. The type on the L1 level is called `ProjectBuildBatchConfigProperty`, so I'm not too worried about this one being called `BatchBuildConfig`.

The name is not a coincidence - it's actually a convention we use for return types (see examples [[1]](https://github.com/aws/aws-cdk/blob/855ce59039a577d142d68720e86d81610edffc64/packages/%40aws-cdk/aws-codebuild/lib/source.ts#L21-L32), [[2]](https://github.com/aws/aws-cdk/blob/855ce59039a577d142d68720e86d81610edffc64/packages/%40aws-cdk/aws-codebuild/lib/artifacts.ts#L12-L17)).

A name ending in `Return` seems a little weird given that, so if you could change it, I would appreciate it 🙂.

Reply:
No problem will update then. Thanks for the examples